HACKING.txt

```diff
@@ -60,27 +52,26 @@ repo, from which you can submit a pull request.
 - Install ``setuptools-git`` into the virtualenv (for good measure, as we're
   using git to do version control):
 
-   $ $VENV/bin/easy_install setuptools-git
+   $ $VENV/bin/pip install setuptools-git
 
-- Install Pyramid from the checkout into the virtualenv using ``setup.py
-  dev``.  ``setup.py dev`` is an alias for "setup.py develop" which also
-  installs testing requirements such as nose and coverage.  Running
-  ``setup.py dev`` *must* be done while the current working directory is the
-  ``pyramid`` checkout directory:
+- Install Pyramid from the checkout into the virtual environment, where the
+  current working directory is the ``pyramid`` checkout directory. We will
+  install Pyramid in editable (development) mode as well as its testing
+  requirements.
 
    $ cd ~/hack-on-pyramid
-   $ $VENV/bin/python setup.py dev
+   $ $VENV/bin/pip install -e .
```

Comment:
You can just make this line `$VENV/bin/pip install -e .[testing,docs]` instead of adding multiple lines.